dev-libs/libsigc++/libsigc++-3.4.0.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit gnome.org flag-o-matic meson-multilib
+
+DESCRIPTION="Typesafe callback system for standard C++"
+HOMEPAGE="https://libsigcplusplus.github.io/libsigcplusplus/
+	https://github.com/libsigcplusplus/libsigcplusplus"
+
+LICENSE="LGPL-2.1+"
+SLOT="3"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~x64-solaris ~x86-solaris"
+IUSE="gtk-doc test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	gtk-doc? (
+		app-doc/doxygen[dot]
+		dev-lang/perl
+		dev-libs/libxslt
+	)
+"
+
+src_prepare() {
+	default
+
+	if ! use test; then
+		sed -i -e "/^subdir('tests')/d" meson.build || die
+	fi
+}
+
+multilib_src_configure() {
+	filter-flags -fno-exceptions #84263
+
+	local emesonargs=(
+		-Dbuild-examples=false
+		$(meson_native_use_bool gtk-doc build-documentation)
+		-Dbuild-tests=false
```

Comment:
I think you forgot to wire this up?

Should be

````diff
+               $(meson_use test build-tests)
````

and drop the whole `src_prepare` function.

Reply:
Thanks, done in https://github.com/gentoo/gentoo/commit/58f95215f62450cbf1daebad9d66bfe6af6edc38